File: varda_dienas.py
import datetime
import sys
import json

try:
	f = open('json.txt', 'r').read()
	v = json.loads(f)

	if len(sys.argv) > 1:
		print ", ".join(v[str(sys.argv[1]).split(".")[1]][str(sys.argv[1]).split(".")[0]])
	else:
		print ", ".join(v[str(datetime.datetime.now().month)][str(datetime.datetime.now().day)])
except:
	"Can't find a names"

File: varda_dienas.rb
require 'rubygems'
require 'hpricot'
require 'open-uri'
require 'date'

doc = open("http://www.vardadienas.lv/search/#{Time.now.day}.#{Time.now.month}") { |f| Hpricot(f) }

names = Array.new

doc.search("#search_results/div[@class='search_row search_row_date']/span[@class='result_name_date']/a").each do |name|
  names << name.inner_html
end

puts names.join(", ")